website/blog/2024/04-23-scrapy-vs-crawlee/index.md

```diff
@@ -0,0 +1,318 @@
+
+Apify Store is a JavaScript rendered website, so we will scrape it in this example using the `scrapy-playwright` integration. 
+
+For installation and to make changes to [`settings.py`], please follow the instructions on the `scrapy-playwright` [repository on GitHub](https://github.com/scrapy-plugins/scrapy-playwright/tree/main?tab=readme-ov-file#installation).
+
+Then, create a spider with this code to scrape the data:
+
+
+```py
+    import scrapy
+
+    class ActorSpider(scrapy.Spider):
+        name = 'actor_spider'
+        start_urls = ['https://apify.com/store']
+
+        def start_requests(self):
+            for url in self.start_urls:
+                yield scrapy.Request(
+                    url,
+                    meta={"playwright": True, "playwright_include_page": True},
+                    callback=self.parse_playwright
+                )
+
+        async def parse_playwright(self, response):
+            page = response.meta['playwright_page']
+            await page.wait_for_selector('.ActorStoreItem-title-wrapper')
+            actor_card = await page.query_selector('.ActorStoreItem-title-wrapper')
+            
+            if actor_card:
+                actor_text = await actor_card.text_content()
+                yield {
+                    'actor': actor_text.strip() if actor_text else 'N/A'
+                }
+
+            await page.close()
+```
+
+
+In Crawlee, you can scrape JavaScript rendered websites using the built-in headless [Puppeteer](https://github.com/puppeteer/puppeteer/) and [Playwright](https://github.com/microsoft/playwright) browsers. It is important to note that, by default, Crawlee scrapes in headless mode. If you don't want headless, then just set `headless: false`.
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+
+<Tabs>
+<TabItem value="javscript" label="Playwright">
+
+```js
+import { PlaywrightCrawler } from 'crawlee';
+
+const crawler = new PlaywrightCrawler({
+    async requestHandler({ page }) {
+        const actorCard = page.locator('.ActorStoreItem-title-wrapper').first();
+        const actorText = await actorCard.textContent();
+        await crawler.pushData({ 
+            'actor': actorText 
+        });
+    },
+});
+
+await crawler.run(['https://apify.com/store']);
+```
+
+</TabItem>
+<TabItem value="js" label="Puppeteer">
+
+```js
+import { PuppeteerCrawler } from 'crawlee';
+
+const crawler = new PuppeteerCrawler({
+    async requestHandler({ page }) {
+        await page.waitForSelector('.ActorStoreItem-title-wrapper');
+        const actorText = await page.$eval('.ActorStoreItem-title-wrapper', (el) => {
+            return el.textContent;
+        });
+        await crawler.pushData({ 
+            'actor': actorText 
+        });
+    },
+});
+
+await crawler.run(['https://apify.com/store']);
+```
+
+</TabItem>
+</Tabs>
+
+
+
```

Comment:
there are several places where you have more than one blank line, it should be only one blank line, always